src/communication/v2/session.hpp

```diff
@@ -366,17 +363,17 @@ class Session final : public std::enable_shared_from_this<Session<TSession, TSes
       : socket_(CreateSocket(std::move(socket), server_context)),
         strand_{boost::asio::make_strand(GetExecutor())},
         output_stream_([this](const uint8_t *data, size_t len, bool have_more) { return Write(data, len, have_more); }),
-        session_{*session_context, endpoint, input_buffer_.read_end(), &output_stream_},
+        session_{session_context->ic,       endpoint, input_buffer_.read_end(), &output_stream_, session_context->auth,
```

Comment:
Before endpoint do we have unnecessary spaces?

Reply:
That is how clang-format does it....maybe we should change our clang-format config. It is not chopping the arguments onto separate lines, but trying to keep endpoint aligned with `session_context->audit_log` inside the ifdef